packages/bridge-ui-v2/src/libs/token/getCrossChainAddress.ts

```diff
@@ -95,6 +95,13 @@ export async function getCrossChainAddress({
 
     // return await destTokenVaultContract.read.canonicalToBridged([BigInt(destChainId), bridgedAddress]);
   } else {
+    if (!(srcChainId in routingContractsMap) || !(destChainId in routingContractsMap[srcChainId])) {
```

Comment:
Nitpick, how about?
```ts
if (!routingContractsMap?.[srcChainId]?.[destChainId]) { ... }
```
does the same job and looks cleaner. Just a suggestion

Reply:
Updated. Thanks